x-pack/plugins/oss_telemetry/server/lib/tasks/visualizations/task_runner.ts

```diff
@@ -68,6 +75,8 @@ async function getStats(callCluster: APICaller, index: string) {
       spaces_min: _.min(spaceCounts),
       spaces_max: _.max(spaceCounts),
       spaces_avg: total / spaceCounts.length,
+      saved_30_days_total: curr.filter((c) => c.past_days <= 30).length,
```

Comment:
NIT: you might want to consider adding `saved_7_days_total` as well. After doing the Application Usage collector, I had a follow-up issue to add the 7-days counter as well (https://github.com/elastic/kibana/issues/59761) 😅

Reply:
Haha ok, will add it ;)